hearing.py

```diff
@@ -763,7 +756,7 @@ def was_defendant_served(self, soup) -> Dict[str, str]:
         served_tags = soup.find_all(text="Served")
         for service_tag in served_tags:
             date_tag = service_tag.parent.find_next_sibling("td")
-            defendant_tag = service_tag.parent.parent.parent.parent.parent.td
```

Comment:
I'm wondering if this will pass the williamson_0 tests. And if not, is the difference because we collected the pages differently, because the Wilco website served them in different formats at different times, or what?

Reply:
Seems like it did.. I collected the page by writing the raw data that python read to a file, that may be the difference